modules/monitoring-modifying-core-platform-alerting-rules.adoc

```diff
@@ -17,6 +17,11 @@ For example, you can change the severity label of an alert, add a custom label,
 .Procedure
 
 . Create a new YAML configuration file named `example-modified-alerting-rule.yaml` in the `openshift-monitoring` namespace.
++
+[IMPORTANT]
+====
+You must create the `AlertRelabelConfig` object in the `openshift-monitoring` namespace. Otherwise, the change to alert label is not accepted.
```

Comment:
Hello @ousleyp thank you for your suggestion. I didn't think about it in that way, but you are 100% right. I am even thinking of removing the `in the openshift-monitoring namespace.`  from the first step as that does not make much sense there, it is most important for the second step. Thank you for pointing that out to me, this shows how important are the fresh pair of eyes here, as i would not think about this otherwise! :pray: :heart:

Reply:
@eromanova97 Thank you so much for the kind words! Your openness to the feedback is worth celebrating as well, as it can be rare in peer review scenarios. :)